web/src/services/loans.service.js

```diff
@@ -109,6 +109,16 @@ const getLoan = (id) => {
         ? Number(BigInt(loan.funded_value_atto_dai) / BigInt(10 ** 18))
         : 0;
 
+      loan.funding_fee = loan.funding_fee_atto_dai_per_dai
+        ? Number(BigInt(loan.funding_fee_atto_dai_per_dai) / BigInt(10 ** 16)) /
```

Comment:
Why 2 divisions?

Reply:
If I divided per `BigInt(10 ** 18)`, the result would be 0, I don't know why. This way the correct result is obtained.

Reply:
Maybe it's related with BigInt